db/schema.rb

```diff
@@ -122,9 +122,6 @@
     t.index ["subject_id"], name: "IX_course_subject_subject_id"
   end
 
-  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
-  end
-
```

Comment:
It might be worth running `rails data:migrate` on your development db just so that this doesn't get removed ... otherwise it's going to keep appearing and disappearing.

Reply:
do i need to run `rails db:migrate` as well as that was what changed `schema.rb`

Reply:
Running `rails data:migrate` should just add this table to your db, and hopefully also dump the schema with it.

Reply:
dump the migration file to the `data` directory and run `rails data:migrate` I think that is right...